classes/scripts/runner.php

```diff
@@ -1170,48 +1239,14 @@ protected function doRun()
 		}
 		else
 		{
-			$arguments = $this->argumentsParser->getValues();
-
-			if (sizeof($arguments) <= 0)
-			{
-				$this->verbose($this->locale->_('Using no CLI argument…'));
-			}
-			else
-			{
-				$this->verbose(sprintf($this->locale->__('Using %s CLI argument…', 'Using %s arguments…', sizeof($arguments)), $this->argumentsParser));
-			}
-
-			if (sizeof($this->configFiles) > 0)
-			{
-				foreach ($this->configFiles as $configFile)
-				{
-					$this->verbose($this->locale->_('Using \'%s\' configuration file…', $configFile));
-				}
-			}
-
-			$autoloaderFile = $this->runner->getAutoloaderFile();
+			$table = new cli\table(array('Property', 'Value'));
```

Comment:
perhaps we can have an interface for that part, with no default implementation (why a default one ?)

and then like the rest create a setVerboseRenderer(verboseRendererInterface $vr)? with a default renderer == table ?